docs/dev/devel-env.rst

```diff
@@ -286,12 +294,157 @@ Also, run the **integration tests** for ListenBrainz.
 When the tests complete, you will see if your changes are valid or not. These tests
 are a helpful way to validate new changes without a lot of work.
 
-FAQ's
+Set up Listenbrainz Spark development environment
+=================================================
+
+The Listenbrainz Spark module is used to generate recommendations based on collaborative
+filtering using Apache Spark. The module is also used to generate stats using BigQuery.
```

Comment:
We are not using BigQuery for stats. We use Apache SPark for stats as well.

Reply:
Okay. Got it.